generator/operation_test.go

```diff
@@ -27,35 +27,11 @@ import (
 func TestUniqueOperationNames(t *testing.T) {
 	doc, err := spec.Load("../fixtures/codegen/todolist.simple.yml")
 	if assert.NoError(t, err) {
-		sp := doc.Spec()
-		sp.Paths.Paths["/tasks"].Post.ID = "saveTask"
-		sp.Paths.Paths["/tasks"].Post.AddExtension("origName", "createTask")
-		sp.Paths.Paths["/tasks/{id}"].Put.ID = "saveTask"
-		sp.Paths.Paths["/tasks/{id}"].Put.AddExtension("origName", "updateTask")
-
-		ops := gatherOperations(doc, nil)
-		assert.Len(t, ops, 4)
-		_, exists := ops["saveTask"]
-		assert.True(t, exists)
-		_, exists = ops["saveTask1"]
-		assert.True(t, exists)
-	}
-}
-
-func TestEmptyOperationNames(t *testing.T) {
-	doc, err := spec.Load("../fixtures/codegen/todolist.simple.yml")
-	if assert.NoError(t, err) {
-		sp := doc.Spec()
-		sp.Paths.Paths["/tasks"].Post.ID = ""
-		sp.Paths.Paths["/tasks"].Post.AddExtension("origName", "createTask")
-		sp.Paths.Paths["/tasks/{id}"].Put.ID = ""
-		sp.Paths.Paths["/tasks/{id}"].Put.AddExtension("origName", "updateTask")
-
```

Comment:
To document what this code regeneration breakage looks like in the former naming convention, here is an example.  This is some handwaving, but you will get the idea.

1. generate a server with the docker spec in the fixture folder
2. look in configureAPI().  notice handlers with names like create1Container and create5Image.  (that's the handwaving.  recalling from memory.)
3. delete the models/ and restapi/ folders
4. regenerate.  This shouldn't touch the server code from gen 1 but will generate new models and operations
5. Try to rebuild.  It will fail because the new handler names have different unique indices.  For example create10Container and create20Image.  This is a mismatch from the names used in configureAPI() from gen 1.